src/UnisonShare/Api.elm

```diff
@@ -189,16 +190,27 @@ session =
 -- NOTIFICATIONS
 
 
-notifications : Account a -> Maybe NotificationStatus -> Endpoint
-notifications account status =
+notifications : Account a -> Maybe NotificationStatus -> PageCursorParam -> Endpoint
+notifications account status paginationCursor =
     let
-        queryParams =
+        statusQueryParams =
             case status of
                 Just s ->
                     [ string "status" (Notification.statusToString s) ]
 
                 Nothing ->
                     []
+
+        paginationQueryParams =
+            case paginationCursor of
+                PrevPage c ->
+                    [ string "prevCursor" (Paginated.cursorToString c) ]
+
+                NextPage c ->
+                    [ string "nextCursor" (Paginated.cursorToString c) ]
```

Comment:
@ChrisPenner these are the query params for cursors that i'm passing along to the API.